timesketch/models/__init__.py

```diff
@@ -106,6 +106,15 @@ def __tablename__(self):
     created_at = Column(DateTime(), default=func.now())
     updated_at = Column(DateTime(), default=func.now(), onupdate=func.now())
 
+    @classmethod
+    def get(cls, **kwargs):
```

Comment:
I opt for removing this. This is a bit confusing with the build in Class.query.get(id). In the code below, just use the query syntax:
```
SearchTemplate.query.filter_by(name=u'foo').first()
```